scipy/linalg/tests/test_misc.py

```diff
@@ -0,0 +1,4 @@
+from scipy.linalg import norm
+
+def test_norm():
```

Comment:
Just noticed one more tiny issue: PEP-8 requires two blank line before a module-level `def`:
```suggestion

def test_norm():
```